cookbook/console/style.rst

```diff
@@ -215,6 +215,13 @@ Progress Bar Methods
         // advances the progress bar 10 steps
         $io->progressAdvance(10);
 
+:method:`Symfony\\Component\\Console\\Style\\StyleInterface::progressRedrawFrequency`
+    It refreshs the progress bar with the given number of steps (or ``1``step
```

Comment:
The comment of your code (`refreshses the progress bar every 2 steps`)  is perfect and easy to understand ... but the description of the method is not as clear as that, specially this part: *"with the given number of steps"*.